include/NovelRT/Utilities/Event.h

```diff
@@ -3,6 +3,7 @@
 
 #include "../Atom.h"
 #include <functional>
+#include <vector>
```

Comment:
Added this as an attempt to fix a probable issue with macOS. But CI randomly decided to *not* run the macOS pipeline...

Reply:
Works now!